src/partials/dialog.settings.html

```diff
@@ -11,9 +11,47 @@ <h2 translate>settings.SETTINGS</h2>
         </md-toolbar>
         <md-dialog-content>
             <div class="md-dialog-content">
+                <section>
+                    <md-subheader class="md-accent"><span translate>settings.notifications.NOTIFICATIONS</span>
+                    </md-subheader>
+                    <md-list flex>
+                        <md-list-item>
+                            <md-checkbox
+                                    ng-disabled="!ctrl.notificationApiAvailable || ctrl.notificationPermission === false"
+                                    ng-model="ctrl.desktopNotifications"
+                                    ng-change="ctrl.setWantsNotifications(ctrl.desktopNotifications)"
+                                    aria-label="Show desktop notifications">
+                            </md-checkbox>
+                            <p translate>settings.notifications.SHOW_NOTIFICATIONS</p>
+                        </md-list-item>
+                        <md-list-item>
+                            <md-checkbox
+                                    ng-disabled="!ctrl.desktopNotifications"
```

Comment:
It finally works as intended. At first i had the string inside `md-checkbox` but it didn´t show, so i put it outside. But according to the docs it shoult work this way. The problem was that angular thougt it would be okay to set the visibilit of the label to `display: none`... 😑
I added a custom CSS rule to show them

Will create a PR in a few